HIP-XXXX.md

```diff
@@ -0,0 +1,298 @@
+that requires zero network lookups and no help from the Handshake network.
+
+#### Looking up a name
+
+We first begin by finding the IP address(s). Any DNS forwarding resolver MAY be used.
+The resolvers are only necessary for performing A/AAAA DNS lookups. They do not need to be trusted.
+It is RECOMMENDED to use a forwarding resolver that encrypts DNS queries. Clients MAY route DNS traffic over
+Tor, VPN, Oblivious DoH Proxy or any other means to protect their privacy. To avoid creating a single
+point of failure, clients MAY rely on several resolvers. To promote decentralization, clients MAY use a random
+resolver from an aggregator. The discovery, aggregation and anonymization of resolvers may be defined by a separate standard.
+
+#### Verifying a certificate
+
+The server certificate MUST contain all the data needed to verify the site's identity. This includes the Urkel proof(s), TLSA record & embedded DNSSEC authentication chain see Servers section for more details. Clients MUST assume the certificate is invalid otherwise [TODO: backward compatibility?]. Refer to the Servers section for the format and extensions ids.
+
+For a certificate to be considered valid, validators MUST find an authentication path starting from one of the stored Urkel
+roots and verifying the matching Urkel proof in the certificate. The proof MUST contain a DS RRSet or a HIP-5 delegation
+to be used as a trust anchor for the TLD. The trust anchor can be used to verify the DNSSEC authentication
+chain in the certificate and validate the TLSA RRSet. Clients can use the validated
+TLSA RRSet to verify DANE as described in RFC6698, RFC7671 and RFC9102.
+
+
+## Servers
+
+For servers to be compliant with this standard they MUST send any proofs & DNSSEC records necessary for clients to complete the TLS Handshake by relying only on the urkel root to prove the trust chain. 
+
+### Building a certificate chain
+
+Example x509 certificate for https://proofofconcept:
+
+```
++------------------------------------------------------------------+
+| End-entity certificate: proofofconcept.                          |
++------------------------------------------------------------------+       
+| Version: 3 (0x2)                                                 |
+| Serial Number: ............                                      |  
+| X509v3 extensions:                                               |
+|   X509v3 Subject Alternative Name:                               |
+|     DNS: proofofconcept                                          |
+|   X509v3 Basic Constraints:                                      |
+|     CA:FALSE                                                     |
+|   Urkel: [                                                       |
+|     Root: 6c3078b26c3f7858d78bd2b63530d66b413f32f0ace083e8f21... |
+|     Proof: <proof>                                               |  
+|     Data: proofofconcept. DS,                                    | 

```

Comment:
May want to mention somewhere that `Data` here is actually an entire serialized `Namestate` object and may be over 512 bytes.

Reply:
I think it's clear that this is an entire urkel proof or two. I will add more to this section once there's a PoS just want to get a hip number/have the draft merged for now.